install.py

```diff
@@ -331,27 +282,28 @@ def dev_reload(target: Path):
 
 # Run
 if __name__ == "__main__":
-	if not webthemedir.exists():
-		raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} Web Theme directory {TEXT_BOLD}{WEB_THEME_DIR}{TEXT_RESET}{TEXT_RED} does not exist. Make sure you're running the installer from its root directory{TEXT_RESET}")
+	if not adwaitadir.exists():
+		raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} Web Theme directory {TEXT_BOLD}{ADWAITA_DIR}{TEXT_RESET}{TEXT_RED} does not exist. Make sure you're running the installer from its root directory{TEXT_RESET}")
 	if not colorthemedir.exists():
 		raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} Color Theme directory {TEXT_BOLD}{COLOR_THEME_DIR}{TEXT_RESET}{TEXT_RED} does not exist. Make sure you're running the installer from its root directory{TEXT_RESET}")
 
 	parser = ArgumentParser(description = "Adwaita-for-Steam installer")
 	parser.add_argument("-c", "--color-theme", default = "adwaita", help = "Choose color theme")
+	parser.add_argument("-cu", "--custom-css", action = "store_true", help = "Enable Custom CSS")
```

Comment:
I don't like multiletter short arguments, especially when there are already other arguments using the same letters.
Also it's often possible to combine short arguments after a single `-`, so `-cu` is even more confusing if it can potentially be equivalent to `-c -u` and if we have both of these options already.
I don't even know how `ArgumentParser` deals with this case.

Maybe now is a good time to remove `-cu`, `-fi`, `-we` and only keep long arguments for them.

`--web-extras` can be renamed to `--extras`/`-e`.